### PR TITLE
Fix MXCall answer not being sent to server in some cases

### PR DESCRIPTION
**Pull request by langleyd**
Resolves https://github.com/vector-im/element-ios/issues/6359

### CODE REVIEW

**Comment by langleyd** [comment]
This is a draft PR as the fix needs to be verified on a particular deployment first.

**Comment by aringenbach** [comment]
The issue has been narrowed down to the call to WebRTC that sets the remoteDescription. This call is actually never made when the issue occurs. 
MXCall notably uses the main operationQueue in order to handle calls which means that something else might have been interfering. Moving all calls operation to a dedicated operationQueue has fixed the issue for the affected users.

File: MatrixSDK/VoIP/MXCall.m [hunk 1]
```diff
@@ -1124,10 +1129,14 @@ - (void)handleCallInvite:(MXEvent *)event
     [callStackCallOperationQueue addOperationWithBlock:^{
         MXStrongifyAndReturnIfNil(self);
         
+        MXLogDebug(@"[MXCall][%@] processing block", self.callId)
```
Comment (ismailgulek) [inline MatrixSDK/VoIP/MXCall.m:1132]: Can we make this comment more explanatory?

**Review by ismailgulek — APPROVED** [review]
One inline comment, otherwise LGTM!

**Review by ismailgulek — APPROVED** [review]

**Pull request merged**